README.md

```diff
@@ -1,70 +1,68 @@
-# Getting Started with Create React App
+![](https://img.shields.io/badge/Microverse-blueviolet)
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+# React capstone project - Metrics webapp
 
-## Available Scripts
+> Single Page Application that fetch data Narrativa COVID API, lists the metrics by city and displays additional information when clicked on a city link.
 
-In the project directory, you can run:
+## Live version
 
-### `yarn start`
+You can access the live version [here](https://ecstatic-shockley-f983ac.netlify.app/)).
 
-Runs the app in the development mode.\
-Open [http://localhost:3000](http://localhost:3000) to view it in the browser.
 
-The page will reload if you make edits.\
-You will also see any lint errors in the console.
+## Video presentation
 
-### `yarn test`
+You can access the video presentation of the project [here](https://youtu.be/BmWjB2lG0WQ).
 
-Launches the test runner in the interactive watch mode.\
-See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
+## Built With
 
-### `yarn build`
+- JavaScript/React
+- React Router
+- React-redux
+- Redux-logger
+- JSX
+- CSS
+- VSCode
+- Webpack
+- Babel
+- Ubuntu 20.04.3
+- Linters
+- Jest
 
-Builds the app for production to the `build` folder.\
-It correctly bundles React in production mode and optimizes the build for the best performance.
+## Setup
 
-The build is minified and the filenames include the hashes.\
-Your app is ready to be deployed!
+- Get the link of the repository: `git@github.com:IjayAbby/react-capstone-covid19-data.git`
+- Clone it as `git@github.com:IjayAbby/react-capstone-covid19-data.git` on a Terminal
```

Comment:
### Optional
- [ ] I think adding the command `git clone` before the link to your repo will make the instructions clearer especially for someone who is not used to version control